### ISSUE TITLE
can not reproduce mAP of vgg16 ssd

### ISSUE THREAD

**Issue by lippman1125** [qfgaohao/pytorch-ssd#70]
I  used default params as you mentioned in README.md

Average Precision Per-class:
aeroplane: 0.7043393616355978
bicycle: 0.7484625112318068
bird: 0.5391020496449674
boat: 0.5449510714961079
bottle: 0.2791507827286398
bus: 0.7562217133144244
car: 0.8028595013771453
cat: 0.7318784859760551
chair: 0.43824672270148696
cow: 0.6129443585685079
diningtable: 0.6424748849405967
dog: 0.690911377338733
horse: 0.7815908558250976
motorbike: 0.7501156480252186
person: 0.7218064368058872
pottedplant: 0.3213606906997066
sheep: 0.6274970730440814
sofa: 0.6874466252243511
train: 0.7814315562894104
tvmonitor: 0.6173514609870225

Average Precision Across All Classes:0.6390071583927421

**Comment by qfgaohao** [comment]
@lippman1125 Have you used a pre-trained image net?

**Comment by dongnana777** [comment]
Is the code of hard negative mining right？

**Comment by lippman1125** [comment]
@qfgaohao why do you use batchnorm instead of L2Norm？

**Comment by qfgaohao** [comment]
@lippman1125 L2Norm was not supported by ONNX at that time. So I replaced it with BatchNorm.  Your params look good. I don't know where the accuracy difference comes from.

**Comment by lippman1125** [comment]
@qfgaohao which metric method do you use, use_voc2007_metric?

**Comment by qfgaohao** [comment]
@lippman1125 yes.

**Comment by lippman1125** [comment]
@qfgaohao I made a mistake in loading pretrained vgg16 model。The code is ok。 Thanks.
